dash-pipeline/SAI/specs/dash_eni.yaml

```diff
@@ -639,6 +639,19 @@ sai_apis:
     valid_only: null
     is_vlan: false
     deprecated: false
+  - !!python/object:utils.sai_spec.sai_attribute.SaiAttribute
+    name: SAI_ENI_ATTR_INBOUND_DASH_ENCAPSULATION
```

Comment:
Hi Marian, may I know which scenario is this? In my understanding, we need to support both encapsulations in the inbound side at the same time, so this solution might not work.